lib/os/fdtable.c

```diff
@@ -72,6 +73,12 @@ static struct fd_entry fdtable[CONFIG_ZVFS_OPEN_MAX] = {
 #endif
 };
 
+#ifdef CONFIG_POSIX_DEVICE_IO_STDIN_STDOUT_STDERR
+FILE *stdin = &fdtable[0];
```

Comment:
I guess we might need to add `FILE` to https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/checkpatch/typedefsfile

Reply:
Maybe - I thought it might have been because the minimal libc defines `stdin`, `stdout`, `stderr` as macros. Hard to say.

Reply:
It seems to fix the issue in vscode locally, created #74507